Libplanet/Assets/FungibleAssetValue.cs

```diff
@@ -0,0 +1,530 @@
+#nullable enable
+using System;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Numerics;
+using System.Runtime.Serialization;
+using Libplanet.Serialization;
+
+namespace Libplanet.Assets
+{
+    /// <summary>
+    /// Holds a fungible asset value which holds its <see cref="Currency"/> together.
+    /// <para>It behaves like numbers except for division operator (<c>/</c>) to prevent to forget
+    /// to handle its remainder; use <see cref="DivRem(FungibleAssetValue)"/> and <see
+    /// cref="DivRem(BigInteger)"/> methods instead.</para>
+    /// </summary>
+    [Serializable]
+    public readonly struct FungibleAssetValue :
+        IEquatable<FungibleAssetValue>,
+        IComparable<FungibleAssetValue>,
+        IComparable,
+        ISerializable
```

Comment:
It would be better if we also adding [`IForamttable`](https://docs.microsoft.com/en-us/dotnet/api/system.iformattable?view=netstandard-2.0). (even next PR).

Reply:
I also wanted to implement `IFormattable` in this patch, but its specification seemed quite complex… I am going to implement it next time